miniapps/electromagnetics/maxwell.cpp

```diff
@@ -479,7 +479,7 @@ void dipole_pulse(const Vector &x, double t, Vector &j)
       j = v;
    }
 
-   j *= a * (t - b) * exp(-0.5 * pow((t-b)/c, 2)) / (c * c);
+   j *= a * (t - b) * exp(-0.5 * pow((t-b)/c, 2.0) / (c * c));
```

Comment:
It looks like the `(c * c)` has been moved inside the exponential function.

Reply:
Oh man 🤦 , thanks Mark